packages/extension-browser/package.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Can you ignore the extension when testing for now until we have a solution?

Reply:
Done.